hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -426,6 +456,39 @@ private boolean isMultiAssignmentsEnabled() {
                 .runAsSystem(() -> tenantConfigurationManagement.getConfigurationValue(key, valueType).getValue());
     }
 
+    private void handleThingCreatedRequest(Message message, String tenant, String virtualHost) {
+        if (message.toString().contains("name")) {
+            handleNameContainedRequest(message, tenant, virtualHost);
```

Comment:
What is in case of message is null?

Reply:
The message was also used before the adaptations to read the message properties and there we also had no additional null check - so I expect this should be fine here.